libraries/standard/mqtt/utest/mqtt_utest.c

```diff
@@ -691,6 +692,17 @@ void test_MQTT_Connect_receiveConnack( void )
     MQTT_DeserializeAck_ExpectAnyArgsAndReturn( MQTTBadResponse );
     status = MQTT_Connect( &mqttContext, &connectInfo, NULL, timeout, &sessionPresent );
     TEST_ASSERT_EQUAL_INT( MQTTBadResponse, status );
+
+    /* Session present is received when a clean session is requested. */
```

Comment:
(Nit): Can be clearer with "Test case when broker sends session present flag in response to a clean session connection request. "